api/routers/main.js

```diff
@@ -6,6 +6,8 @@ const csrfProtection = require('../policies/csrfProtection');
 router.get('/', MainController.home);
 router.get('/case-studies/', MainController.examples);
 router.get('/content/examples', (req, res) => res.redirect('/case-studies'));
+router.get('/contact/', MainController.contact);
+router.get('/content/contact', (req, res) => res.redirect('/contact'));
```

Comment:
I don't think we need this; the other line was to keep old links compatible with the new site structure.